app/common/util/src/main/java/io/syndesis/common/util/ErrorCategory.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.common.util;
+
+public class ErrorCategory {
+
+	public static final String SQL_CONNECTOR_ERROR        = "SQL_CONNECTOR_ERROR";
+	public static final String SQL_DATA_ACCESS_ERROR      = "SQL_DATA_ACCESS_ERROR";
+	public static final String SQL_ENTITY_NOT_FOUND_ERROR = "SQL_ENTITY_NOT_FOUND_ERROR";
+
+	public static final String TWITTER_CONNECTOR_ERROR    = "TWITTER_CONNECTOR_ERROR";
+	public static final String SALESFORCE_CONNECTOR_ERROR = "SALESFORCE_CONNECTOR_ERROR";
+	public static final String GMAIL_CONNECTOR_ERROR      = "GMAIL_CONNECTOR_ERROR";
+	public static final String KAFKA_CONNECTOR_ERROR      = "KAFKA_CONNECTOR_ERROR";
```

Comment:
I kinda like having it be more specific, especially when a user is trying to fix something. BTW we don't really need this class, the errors are defined in the 'standardizedErrors' section in each connector json. Seems like it'd be nice to have a complete list for accounting purposed though. But if updating is an issue then we could do away with this class.

Reply:
Yeah I feel it would be, I'd like us to be in a situation where the connectors available to Syndesis are dynamic, and to that end we don't or can't know which connector is present in an environment.

Reply:
I've split up the ErrorCategory, so we have one per connector, plus the one common one.